### ISSUE TITLE
Thanks!

### ISSUE THREAD

**Issue by coderschoolreview** [Zincate/Homeworksubmission#1]
## Goal of this Assignment
The goal of this assignment was to introduce you to 2 main concepts in Machine Learning: 
- Data Visualization
- Exploratory Data Analysis. 

You learn how to query and clean data using pandas library in Python, make some plots which help to understand more about data with Seaborn library.

## Things you did well:
- You accomplished almost the whole assignment. This is a very great start !
- You can make use of Pandas functions and Python code very well.
- Your solution is comprehensive and it helps me a lot when grading your work!

## Things to work on:
- What you can improve is trying to accomplish all questions in this homework, as it is fundamental for upcoming sections.
- I believe that you can complete all this questions easily if you can practice more on Pandas :D
- Also, reference Pandas documentations more to understand well how it works .

## To sum up
- Overall, excellent work! You are demonstrating that you are understanding the material and doing a great job of applying it. Keep it up!
- Keep the spirit with homework assignments and you will improve very fast :D

**Comment by coderschoolreview** [comment]
The goal of this assignment is for you to learn to build and evaluate a model using scikit-learn library in Python. You learn how to do Data pre-processing, splitting it into training and test sets, training your model using training sets and evaluating its performance on test sets.

Things you did well: 
- Discovering the usage of many new functions!
- Understanding how to use data to train a model using library in Python. This is a very important step!

Some minor tips:
- The Mean Absolute Error(MAE), Mean Squared Error(MSE), Root Mean Squared Error(RMSE) could be used like below: metrics.meanabsoluteerror(y_test, predictions) metrics.meansquarederror(y_test, predictions) np.sqrt(metrics.meansquarederror(y_test, predictions)) 
- In the “Residual” section, you count the different between your y_test and predictions, and then put it into graph to see whether it is a normal distribution. So: residuals = y_test-predictions, and then plot it using: sns.distplot(residuals,bins=10)

**Comment by coderschoolreview** [comment]
Hi Chikara Miyake,

Good work on your Assignment 3. Now you are confident to perform tasks that include sentiment analysis.

One minor tip:

- You could delete this function 

"def tokenizer(text):
    return text.split()"

since you didn't use them your assignment.
